lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -404,6 +404,21 @@ export class AuthorizationCodeClient extends BaseClient {
             parameterBuilder.addExtraQueryParameters(request.extraQueryParameters);
         }
 
+        if (request.nativeBroker) {
```

Comment:
This is also present on line 422 - can we combine?

Reply:
Done, merge issue. Thanks for catching this.